### ISSUE TITLE
Live example of Tour of Heroes isn't adding Heroes

### ISSUE THREAD

**Issue by limakayo** [angular/angular#8740]
**Steps to reproduce and a minimal demo of the problem**

_What steps should we try in your demo to see the problem?_

Go to https://angular.io/docs/ts/latest/tutorial/ click on the "Run the live example" link, in Plunker try add new heroes.

**Current behavior**

`<my-hero-detail (close)="close($event)"></my-hero-detail>` isn't showing after click "Add new hero" button

**Expected/desired behavior**

After click the button "Add new Hero" show `<my-hero-detail>`

**Other information**

Add `HeroDetailComponent` to `HeroesComponent` directives.
Plunker:  http://plnkr.co/edit/8uMwR2AfJ8B77TUGAUSP

**Comment by michalstepien** [comment]
I have error:
Error: Error: XHR error (404) loading https://npmcdn.com/angular2-in-memory-web-api/core.js(…)

**Comment by zoechi** [comment]
Doc issues should be reported in https://github.com/angular/angular.io

**Comment by chrisjordanme** [comment]
I also encountered the error @michalstepien is reporting.  I was able to resolve this by replacing line 5 of main.ts with:

`import { InMemoryBackendService, SEED_DATA } from 'angular2-in-memory-web-api/src/in-memory-backend.service';
`
There might be a better change that should be made in system.config but this resolved my error. I will investigate and then look at reporting the error in the docs or submitting the doc update.
